.github/workflows/ci-python-sdk.yaml

```diff
@@ -384,3 +385,41 @@ jobs:
     env:
       TWINE_USERNAME: __token__
       TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+
+  Run-Benchmark:
+    if: github.event.label.name == 'benchmark'
```

Comment:
@pankajastro Can we change this to run-benchmark? We might use the benchmark to categorize other tickets down the line.